pkgs/development/tools/misc/texinfo/default.nix

```diff
@@ -0,0 +1,110 @@
+{
+  lib,
+  stdenv,
+  buildPackages,
+  updateAutotoolsGnuConfigScriptsHook,
+  fetchurl,
+  perl,
+  xz,
+  libintl,
+  libiconv,
+  bash,
+  gnulib,
+  gawk,
+  freebsd,
+  ncurses,
+  procps,
+  # we are a dependency of gcc, this simplifies bootstraping
+  interactive ? false,
+}:
```

Comment:
```
{ lib
, stdenv
, ...
```

Reply:
[See my comment above](https://github.com/NixOS/nixpkgs/pull/289690#discussion_r1623569633) about nixfmt

Reply:
See the link to "Code conventions" above.